File: LanguAI.Frontend/src/api/models/profile-page-data-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { CardListViewModel } from '../models/card-list-view-model';
import { OtherUserViewModel } from '../models/other-user-view-model';
import { UserViewModel } from '../models/user-view-model';
export interface ProfilePageDataViewModel {
  cardList?: Array<CardListViewModel> | null;
  friendList?: Array<OtherUserViewModel> | null;
  user?: UserViewModel;
}

File: LanguAI.Backend/LanguAI.Backend/ViewModels/User/ProfilePageDataViewModel.cs
using LanguAI.Backend.ViewModels.Card;

namespace LanguAI.Backend.ViewModels.User;

public class ProfilePageDataViewModel
{
    public UserViewModel User { get; set; }
    public List<OtherUserViewModel> FriendList { get; set; }
    public List<CardListViewModel> CardList { get; set; }
}
